content/documentation/pages/5-feature-guides/2-batch/5-restarting.md

```diff
@@ -0,0 +1,25 @@
+---
+path: 'feature-guides/batch/restarting/'
+title: 'Restarting Batch Jobs'
+description: 'Learn how to restart Batch Jobs'
+---
+
+# Batch Job Restart
+
+Spring Cloud Data Flow allows a user to restart a Spring Batch Job. Meaning that if a Spring Batch Job fails, it can be restarted and pick up the work from where it left off. In this section, we will show you how you can restart batch jobs using Spring Cloud Data Flow.
```

Comment:
```suggestion
Spring Cloud Data Flow allows a user to restart a Spring Batch Job. For instance, if a Spring Batch Job fails to run, it can be restarted from SCDF Dashboard, so the batch-job can pick up the work from where it left off. In this section, we will show how you can restart batch-jobs.
```